### PR TITLE
LRCI-225 Wrap LiferaySelenium invocations with a timeout & simplify exception stacktrace

### PR DESCRIPTION
**Pull request by pyoo47**
https://issues.liferay.com/browse/LRCI-225

### CODE REVIEW

**Comment by pyoo47** [comment]
ci:test

File: poshi-runner/src/main/java/com/liferay/poshi/runner/PoshiRunnerExecutor.java [hunk 1]
```diff
@@ -1018,7 +1018,7 @@ else if (i == 2) {
 
 		method = clazz.getMethod(
 			selenium,
-			parameterClasses.toArray(new Class<?>[parameterClasses.size()]));
```
Comment (pyoo47) [inline poshi-runner/src/main/java/com/liferay/poshi/runner/PoshiRunnerExecutor.java]: @hhuijser SF didn't catch the need for this change.

**Comment by kenjiheigel** [comment]
ci:close

**Comment by brianchandotcom** [comment]
Merged. Thx.

**Comment by brianchandotcom** [comment]
@pyoo47 @kenjiheigel see 4a3e532310413d7db3aae6070ce77b34cca711ff

**Pull request closed without merging**